### PR TITLE
Web app contribution guide

### PR DESCRIPTION
**Pull request by Gordonby**
## PR Summary

I've enhanced the contribution guide for the web app with a walkthrough.

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not **Work in Progress**

### CODE REVIEW

**Comment by Gordonby** [comment]
Putting this on @liammoat radar.

File: docs/ContributingWizard.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+The web app is preset driven, different presets will change the values of the controls in the tabs. Most controls map directly to a parameter for the main.bicep code file.
+
+The UI is initially populated with values from the config.json file. Selecting a preset then overlays specific values in the UI.
+
+![overlay preset json](images/wiz-overlay-config.png)
+
+### Guidance in the UI
+
+Whilst not being a teaching resource, adding UI hints, validation and links to the Azure documentation are all  important to the configuration experience.
```
Comment (liammoat) [inline docs/ContributingWizard.md:19]: - [ ] Double space
- [ ] Is it worth mentioning that we want hints to be brief, with a preference for links to documentation?

File: docs/ContributingWizard.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+1. Cluster
+1. AddOns
+1. Networking
+1. Sample Apps
+
+## 2. Default value
+
+Open the [default control config file](../helper/src/config.json), and locate the section corresponding to the tab (eg. Networking). Add a new key value pair, using the identical parameter name from the bicep and the **same default**.
+
+> Choosing the least obtrusive default value is the generally the best tact.
```
Comment (liammoat) [inline docs/ContributingWizard.md:39]: "default value is generally the best tact."

**Review by liammoat — COMMENTED** [review]
In general, looks great. Couple of typos and a suggestion on the hint section.

**Comment by Gordonby** [comment]
Incorporated you feedback @liammoat 👍

**Comment by Gordonby** [comment]
Incorporated your feedback @khowling 👍

**Review by khowling — APPROVED** [review]
brilliant, thank you

**Pull request merged**